pkg/frontend/openshiftcluster_preflightvalidation.go

```diff
@@ -52,9 +52,9 @@ func (f *frontend) preflightValidation(w http.ResponseWriter, r *http.Request) {
 			continue
```

Comment:
Won't all subsequent validation fail if we continue here?

Reply:
Not necessarily as we are unmarshalling each resource separately. If one fails to unmarshall it doesn't mean the rest will, and according to preflight, is not considered a validation failure either.